types/server/typings/options.d.ts

```diff
@@ -42,12 +42,15 @@ export interface CsurfOptions {
     sessionKey?: string | undefined;
 }
 
+export {}
```

Comment:
This shouldn't be here.

Reply:
@typescript-bot told me to put it there. Otherwise EngineArgs is exported automatically.

Reply:
Could add a comment explaining why it's there?